source/iopipe/json/serialize.d

```diff
@@ -1331,7 +1331,7 @@ void deserializeObject(T, JT, Policy)(
 }
 
 void deserializeImpl(T, JT, Policy)(
-    ref Policy policy,
+    auto ref Policy policy,
```

Comment:
While I am a bit hesitant over changing this for the Impl functions, it should be changed consistently if it's going to be changed.

I kind of feel like the change to auto ref here is wrong. You should by default just keep passing down the policy you got. A user is going to be confused if a policy gets swapped out at some point, I wouldn't expect this to be the norm. And of course, it's still possible even if it's ref.

Do you have any use cases to show this is worth it?